Software 0.30 base PR (#1041)
astronomer/docs: software/manage-platform-users.md

<!--- Version-specific -->

> First, take a look at our default roles and permissions in the [default Houston API configuration](https://github.com/astronomer/docs/tree/main/software_configs/0.30/default.yaml) and identify two things:

Reviewer: ```suggestion
Review the default roles and permissions in the [default Houston API configuration](https://github.com/astronomer/docs/tree/main/software_configs/0.30/default.yaml) and determine the following:
```

> 1. What role do you want to configure? (e.g. `DEPLOYMENT_EDITOR`)

Reviewer: ```suggestion
- What role do you want to configure? For example, `DEPLOYMENT_EDITOR`.
```

> 2. What permission(s) would you like to add to or remove from that role? (e.g. `deployment.images.push`)

Reviewer: ```suggestion
- What permission(s) would you like to add to or remove from that role? For example, `deployment.images.push`.
```

No user is assigned the System Editor or Viewer Roles by default, but they can be added by System Admins via our API. Once assigned, System Viewers, for example, can access both Grafana and Kibana but don't have permission to delete a Workspace they're not a part of.

> All three permission sets are entirely customizable on Astronomer Software. For a full breakdown of the default configurations attached to the System Admin, Editor and Viewer Roles, refer to the [Houston API source code](https://github.com/astronomer/docs/tree/main/software_configs/0.30/default.yaml).

Reviewer: ```suggestion
You can customize all three Astronomer Software permission sets to meet your specific requirements. For more information about the default configurations attached to the System Admin, Editor and Viewer Roles, see the [Houston API source code](https://github.com/astronomer/docs/tree/main/software_configs/0.30/default.yaml).
```

> For guidelines on assigning users any System Level role, read below.

Reviewer: Remove this text.
